website/migrations/0259_make_email_unique.py

```
@@ -0,0 +1,104 @@
+def reverse_migration(apps, schema_editor):
+    """
+    Reverse migration - cannot restore deleted users, so this is a no-op.
+    """
+    pass
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("website", "0258_add_slackchannel_model"),
+        ("auth", "0012_alter_user_first_name_max_length"),
+    ]
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)